spec/rubocop/cop/rspec/factory_girl/create_list_spec.rb

```diff
@@ -0,0 +1,76 @@
+RSpec.describe RuboCop::Cop::RSpec::FactoryGirl::CreateList do
+  subject(:cop) { described_class.new }
+
+  it 'flags usage of n.times with no arguments' do
+    expect_offense(<<-RUBY)
+      3.times { create :user }
+      ^^^^^^^ Prefer create_list.
+    RUBY
+  end
+
+  it 'flags usage of n.times when FactgoryGirl.create is used' do
+    expect_offense(<<-RUBY)
+      3.times { FactoryGirl.create :user }
+      ^^^^^^^ Prefer create_list.
+    RUBY
+  end
+
+  it 'ignores create method of other object' do
+    expect_no_offenses(<<-RUBY)
+      3.times { SomeFactory.create :user }
+    RUBY
+  end
+
+  it 'ignores create in other block' do
+    expect_no_offenses(<<-RUBY)
+      allow(User).to receive(:create) { create :user }
+    RUBY
+  end
+
+  it 'ignores n.times with argument' do
+    expect_no_offenses(<<-RUBY)
+      3.times { |n| create :user, created_at: n.days.ago }
+    RUBY
+  end
+
+  it 'ignores n.times when there is no create call inside' do
+    expect_no_offenses(<<-RUBY)
+      3.times { do_something }
+    RUBY
+  end
+
+  it 'ignores n.times when there is other calls but create' do
+    expect_no_offenses(<<-RUBY)
+      used_passwords = []
+      3.times do
+        u = create :user
+        expect(used_passwords).not_to include(u.password)
+        used_passwords << u.password
+      end
+    RUBY
+  end
+
+  it 'ignores FactoryGirl.create calls with a block' do
+    expect_no_offenses(<<-RUBY)
+      3.times do
+        create(:user) { |user| create :account, user: user }
```

Comment:
But why? It could be `create_list(:user, 3) { |user| create :account, user: user }`

Reply:
The documentation doesn't mention this

Reply:
It's [in the spec](https://github.com/thoughtbot/factory_bot/blob/c716ce01b448ce4e0bf855c5a2c63ecb9206322e/spec/acceptance/create_list_spec.rb#L45). Tried in the console, works as well.

Reply:
10x. Will handle it